public_html/profiles/social/themes/socialbase/components/contrib/bootstrap/_utilities.scss

```diff
@@ -52,5 +52,4 @@
 
 .affix {
   position: fixed;
-  top: $navbar-height;
```

Comment:
This happened when I executed gulp init. New version of bootstrap components maybe? Dunno whether this will harm us, but I don't know if we even use this class.